### ISSUE TITLE
Unable to open menu items with VoiceOver on

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#197]
While VoiceOver is on and the user is not able to open menu options "Tap for menu". It takes the user to "enter your 8-digit number" screen.

It should take user to the menu.

iOS 13.5.1 on iPhone 7

![covidalert2](https://user-images.githubusercontent.com/14280062/85878240-6549a200-b7a6-11ea-8b36-be2a0f07b8b4.jpg)
![covidalert3](https://user-images.githubusercontent.com/14280062/85878241-65e23880-b7a6-11ea-84c4-04f1d3dbf0cb.jpg)

**Comment by JuliannaR** [comment]
This was fix in V11. @qanitah can you update testflight and retest?

**Comment by JuliannaR** [comment]
Merged into this issue: https://github.com/cds-snc/covid-shield-mobile/issues/432
